pages/app-layout/runtime-drawers-with-updates.page.tsx

```diff
@@ -128,6 +128,7 @@ export default function WithDrawers() {
       tools={<Info helpPathSlug={helpPathSlug} />}
       toolsOpen={isToolsOpen}
       toolsHide={!hasTools}
+      navigationHide={true}
```

Comment:
why is this needed?

Reply:
The inner content width slightly shrinking because of the vertical scrollbar was affecting the test, and since the side navigation in this dev page is empty (nothing passed to the `navigation` slot) and not relevant for any test performed on this dev page (also visual regression tests), hiding it to free space was more convenient and less reliant on magic numbers than accommodating the viewport width or other measurements.